packages/terra-toolbar/src/Toolbar.module.scss

```diff
@@ -0,0 +1,50 @@
+@import '~terra-mixins/lib/Mixins';
+
+:local {
+  .toolbar {
+    align-items: flex-end;
```

Comment:
are items always expected to vertically align to the bottom?

Reply:
Yes